Staging/EnableGop/EnableGopDirect.inf

```diff
@@ -12,7 +12,7 @@
   BASE_NAME                      = EnableGopDirect
   FILE_GUID                      = 3FBA58B1-F8C0-41BC-ACD8-253043A3A17F
   MODULE_TYPE                    = DXE_DRIVER
-  VERSION_STRING                 = 1.1
+  VERSION_STRING                 = 1.2
```

Comment:
As above. Though I am planning that these will always match, between the two alternative versions (e.g. even if a new feature was only added to one of them).